mp/src/game/server/momentum/mom_triggers.h

```diff
@@ -291,40 +372,63 @@ class CTriggerTrickZone : public CBaseMomZoneTrigger
     CNetworkVar(int, m_iDrawState);
 };
 
+enum TeleportMode
```

Comment:
I'd say follow the naming scheme I see for enums
```suggestion
enum TeleportMode_t
```